### PR TITLE
Add CreateOrUpdateOrganization

### PR DESCRIPTION
**Pull request by jredl-va**
In order to make managing organizations a little easier I have added support for the Create or Update Organization endpoint: https://developer.zendesk.com/rest_api/docs/core/organizations#create-or-update-organization

### CODE REVIEW

**Comment by jredl-va** [comment]
@eggman64 saw you had a recent commit here so figured I would ping you for a review.

File: zendesk/ticket.go [hunk 1]
```diff
@@ -90,13 +91,32 @@ func (c *client) BulkUpdateManyTickets(ids []int64, ticket *Ticket) error {
 	return err
 }
 
+// ListOrganizationTickets list tickets for an organization
+//
+// Zendesk Core API docs: https://developer.zendesk.com/rest_api/docs/core/tickets#list-tickets
+func (c *client) ListOrganizationTickets(organizationID int64, options *ListOptions) ([]Ticket, error) {
+	params, err := query.Values(opts)
```
Comment (eggman64) [inline zendesk/ticket.go]: `params, err := query.Values(options)` with *options* not *opts*

**Comment by eggman64** [comment]
Thanks for your contribution! :)
I've left one comment, please fix. Also, if you could squash your contribution into a single commit, that would be appreciated.

**Comment by jredl-va** [comment]
@eggman64 thanks, will adjust and reopen.

**Pull request closed without merging**